R/shiny.R

```diff
@@ -1348,10 +1348,11 @@ ShinySession <- R6Class(
     registerDataObj = function(name, data, filterFunc) {
       # abusing downloads at the moment
       self$downloads$set(name, list(data = data, filter = filterFunc))
-      return(sprintf('session/%s/dataobj/%s?w=%s',
+      return(sprintf('session/%s/dataobj/%s?w=%s?timestamp=%s',
```

Comment:
You've added a second `?` here--you want `&`.

Also, instead of timestamp, I'd personally use `createUniqueId(8)` or something like that, but I don't have a rational justification for it. :) So either way, just giving you the option.